commons-web/src/main/java/eu/europeana/api/commons/service/authorization/BaseAuthorizationService.java

```diff
@@ -163,6 +167,53 @@ private boolean isOperationAuthorized(String operation, List<GrantedAuthority> a
 	return false;
     }
 
+    /**
+     * This method compares If-Match header with the current etag value.
+     * 
+     * @param etag    The current etag value
+     * @param request The request containing If-Match header
+     * @throws HttpException
+     */
+    public void checkIfMatchHeader(String etag, HttpServletRequest request) throws HttpException {
```

Comment:
this method must not be implemented in authorization service, it belogns to BaseRestControler

Reply:
moved to BaseRestController